a2x.py

```diff
@@ -219,11 +219,11 @@ def shell(cmd, raise_error=True):
         die('failed: %s: %s' % (cmd, e))
     stdoutdata, stderrdata = popen.communicate()
     if OPTIONS.verbose:
-        print(stdoutdata)
-        print(stderrdata)
+        print(stdoutdata.decode('utf-8'))
+        print(stderrdata.decode('utf-8'))
```

Comment:
I'm not sure stdout and stderr are always 'utf-8', being terminal outputs they probably are in locale encoding.

Reply:
Fair enough. Used `universal_newlines=True` to handle decoding within subprocess instead of explicitly doing it outside of it.